roles/common/tasks/main.yml

```diff
@@ -91,6 +90,14 @@
   loop_control:
     loop_var: extra_key
 
+- name: Add ssh config
+  import_tasks: ssh_config.yml
+  when: ssh_config_hosts is defined
+
+- name: Add ssh config
```

Comment:
this duplicates the above task

Reply:
oops. new commit removes the second copy